### ISSUE TITLE
Doesn't effect email content, they're still scorching white. Replaces background image with flat color

### ISSUE THREAD

**Issue by mzso** [EvHaus/google-redesigned#12]
Hi!

My main concer about this and gmail's default themes is that they don't apply to email content they're still black on white.

Also the background image is covered with flat colors. But since I already have a dark background via the image, the flat color is not an improvement, it's just less appealing.

**Comment by EvHaus** [comment]
I agree with you, it's not ideal. However after several attempts I wasn't able to find a solution that would work well for all cases.

Emails will often contain HTML content (ie. newsletters and brochures) which assume a white background. If I change the background to something dark many email become unreadable because they force their own font color and there is no way to replace it, at least not without potentially corrupting the content of the email.

**Comment by mzso** [comment]
[**@EvHaus**](https://github.com/EvHaus) commented on [2022. nov. 1. 15:16 CET](https://github.com/EvHaus/google-redesigned/issues/12#issuecomment-1298576794 "2022-11-01T14:16:28Z - Replied by Github Reply Comments"):
> I agree with you, it's not ideal. However after several attempts I wasn't able to find a solution that would work well for all cases.
> 
> Emails will often contain HTML content (ie. newsletters and brochures) which assume a white background. If I change the background to something dark many email become unreadable because they force their own font color and there is no way to replace it, at least not without potentially corrupting the content of the email.

Having it darker, around medium brightness would still be nice.

**Comment by anthonyrabiaza** [comment]
I added the following to have at least the frame in dark mode:

`    .qd {
        color: #afb4b7;
    }
    
    .gs {
        background-color: #303030;
    }
    
    .aju {
        background-color: #303030;
    }
    
    .adn {
        background-color: #303030;
    }
    
    .nH {
        background-color: #303030;
    }`

**Comment by EvHaus** [comment]
That had no effect for me. However, you could do:

```css
.a3s { filter: invert(1) }
.a3s img { filter: invert(1) }
```

But that will distort colors fairly badly.

**Comment by AvgZing** [comment]
This is my solution.
`.AO,.G-atb {
    margin: auto !important;
    width:80%;
}
.hx, .ha {
    -webkit-filter: invert(87%);
}

.nH.a98.iY {
    border-radius: 10px
}

.aCi, .CToWUd{
-webkit-filter: invert(113%);
}

.bkK>.nH {
    background-color: Transparent;
}

.zA {
    background-color: rgba(32, 32, 32, .8)
}

.iY {
    background-color: #212121
}`
